### ISSUE TITLE
Set travis-ci with a daily cron-job to detect breakage

### ISSUE THREAD

**Issue by gnzlbg** [SkylerLipthay/interpolate_idents#41]
Would it be possible to set up travis-ci with a daily cron-job and email notification on breakage ?

The library is broken again:

```
--> /cargo/registry/src/github.com-1ecc6299db9ec823/interpolate_idents-0.2.7/src/parser_any_macro.rs:12:5
   |
12 |     OneVector,
   |     ^^^^^^^^^ no `OneVector` in the root
```

**Comment by SkylerLipthay** [comment]
This sounds wonderful. I have very little experience with CI in general. Do you care to take the lead? If you are interested, let's make you a collaborator for this repo and add you as an owner on crates.io. I don't use have any use for this crate personally, so I'm happy to turn the maintenance over. No pressure. Let me know!

**Comment by gnzlbg** [comment]
We strongly depend on this crate for `packed_simd`, so I can take care of setting CI for this and can help with keeping CI green as things change in rustc. It would be awesome if you could still give feedbacks about PRs though.

**Comment by SkylerLipthay** [comment]
That's fantastic. I'll maintain co-ownership and I'll subscribe to the CI, so I'll still receive notifications and comment/contribute where I can.

The reason I don't contribute to this library these days has to do with how infrequently I work with nightly Rust. Because of my job I've been swept into the stable world and I'm unfortunately quite behind on the latest compiler developments. So my insights will be limited but I'll do my best.

**Comment by gnzlbg** [comment]
Are you on IRC / discord ? You have to at least do the initial set-up for travis-ci and appveyor.

**Comment by SkylerLipthay** [comment]
Sorry for the delayed response. Yes please add me on discord `sky#9561`

**Comment by gnzlbg** [comment]
Closed by #44 .
